src/components/Music/Music.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import s from "./Music.module.css";
+
+const Music = () => {
+    return (
+     <div>
+       Music
```

Comment:
Consider adding the I18N to all the components.

Reply:
Instead of a string in this section there will be a music player or (mb) music list